### ISSUE TITLE
[Bug] Time Crisis 4 (from Time Crisis Razing Storm collection) gets stuck in the end of "Building SPU cache" screen

### ISSUE THREAD

**Issue by Johnramirezwilson** [RPCS3/rpcs3#15474]
### Quick summary

When Time Crisis 4 (from Time Crisis Razing Storm collection) is loading, it gets stuck in the end of "Building SPU cache" screen.

### Details

When using PS Move, 50 fps limit and Write Color Buffer is ON, Time Crisis 4 (from Time Crisis Razing Storm collection) gets stuck in the end of "Building SPU cache" screen. Deleting the cache will make the game start, but it is impractable to wait loading time everytime.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/15017781/RPCS3.log.gz)

### System configuration

Windows 11
RTX 3050
i7 11800h
8gb RAM

**Comment by marcosgaming10** [comment]
Hi, how to fix this problem??

**Comment by Tovarichtch** [comment]
I reproduce this issue since few builds already, on Linux. Dev branch. Last tested on `0.0.32-16518`

**Comment by AniLeo** [comment]
If it's a regression, it needs to be reported as such, otherwise the chances of the ticket being picked up are low.

Needs to follow the regression issue template instead if that's the case.

**Comment by Johnramirezwilson** [comment]
> If it's a regression, it needs to be reported as such, otherwise the chances of the ticket being picked up are low.
> 
> Needs to follow the regression issue template instead if that's the case.

Didn't know it was a regression, it was my first time trying to play this game.

**Comment by scju** [comment]
I have got this issue too, and I have to remove the SPU cache every time to get into the game.

**Comment by Tovarichtch** [comment]
Issue seems fixed. Tested on `0.0.32-16614`.

**Comment by Tovarichtch** [comment]
Nope, the issue is still there. I've just deleted the cache, I was wrong.

I'll make a proper report when I get the time.

**Comment by vitorary** [comment]
> Nope, the issue is still there. I've just deleted the cache, I was wrong.
> 
> I'll make a proper report when I get the time.

Thanks.

**Comment by MarkPremier** [comment]
Is there an update with this issue? Unsure if it's  been fixed.

**Comment by Tovarichtch** [comment]
> ###
> **[MarkPremier](/MarkPremier) ** commented [Dec 22, 2024](#issuecomment-2558363414)

Issue is still there. I didn't make a full report on this yet because I'm trying to find which commit is causing this. It's taking me forever.

In short: 

- if you enable PS Move and launch the game for the first time, it loads. 
- If you enable PS Move and launch the game for the second/third/fourth/etc time, it gets stuck at 99% of SPU cache compiling.
- If you disable PS Move and launch the game, it loads.

The only workaround for now is the clean the SPU cache everytime you want to play the game after you've launched it. It seems to be only concerned with Time Crisis Razing Storm bundle (which contains 3 games).
